backend/src/Designer/Controllers/PreviewController.cs

```diff
@@ -120,7 +120,7 @@ public async Task<ActionResult<ApplicationSettings>> ApplicationSettings(string
         /// <returns>layoutsets file, or a notfound response if app does not use layoutsets</returns>
         [HttpGet]
         [Route("api/layoutsets")]
-        public async Task<ActionResult<LayoutSets>> LayoutSets(string org, string app)
+        public async Task<ActionResult<JsonNode>> LayoutSets(string org, string app)
```

Comment:
It's changed to JsonNode but method GetLayoutSets returns "LayoutSets" class.
Those 2 should be alligned